src/main/java/trackitnus/storage/JsonAdaptedContact.java

```diff
@@ -74,21 +74,25 @@ public Contact toModelType() throws IllegalValueException {
         }
         final Name modelName = new Name(name);
 
+        final Phone modelPhone;
         if (phone == null) {
-            throw new IllegalValueException(String.format(MISSING_FIELD_MESSAGE_FORMAT, Phone.class.getSimpleName()));
+            modelPhone = null;
+        } else {
+            if (!Phone.isValidPhone(phone)) {
+                throw new IllegalValueException(Phone.MESSAGE_CONSTRAINTS);
+            }
+            modelPhone = new Phone(phone);
```

Comment:
Can we shorten this to 1 `else if {` instead of `else { if {` ?

Reply:
Yeah I think we can!